### PR TITLE
Update relayer api schemas

### PR DESCRIPTION
**Pull request by BMillman19**
* Schema updates to reflect changes in this PR: https://github.com/0xProject/standard-relayer-api/pull/9

### CODE REVIEW

File: schemas/relayer_api_fees_payload_schema.ts [hunk 1]
```diff
@@ -2,12 +2,22 @@ export const relayerApiFeesPayloadSchema = {
     id: '/RelayerApiFeesPayload',
     type: 'object',
     properties: {
+        exchangeContractAddress: {$ref: '/Address'},
+
```
Comment (fabioberger) [inline schemas/relayer_api_fees_payload_schema.ts]: We don't add newlines in any of our schemas, let's remove these.

File: schemas/relayer_api_order_payload_schema.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const relayerApiOrderPayloadSchema = {
+    id: '/RelayerApiOrderPayload',
+    type: 'object',
+    properties: {
+        signedOrder: {$ref: '/SignedOrder'},
```
Comment (fabioberger) [inline schemas/relayer_api_order_payload_schema.ts]: Why is the order nested under the `signedOrder` key?

File: schemas/relayer_api_fees_payload_schema.ts (hunk 1 above)
Comment (BMillman19) [reply schemas/relayer_api_fees_payload_schema.ts]: I was falling the example of order_schemas.ts, I'm happy to remove the newlines from both

File: schemas/relayer_api_order_payload_schema.ts (hunk 2 above)
Comment (BMillman19) [reply schemas/relayer_api_order_payload_schema.ts]: this way we can add new fields to the response without breaking signedOrder json parsing

**Pull request merged**